### PR TITLE
Release 2.9.13-beta2

### PR DESCRIPTION
**Pull request by tidy-dev**
## Description
Looking for the PR for the upcoming second beta of the v2.9.13 series? Well, you've just found it, congratulations!

## Release checklist

- [x] Check to see if there are any errors in Sentry that have only occurred since the last production release
- [x] Verify that all feature flags are flipped appropriately
PR Review notifications still flagged to beta
- [x] If there are any new metrics, ensure that central and desktop.github.com have been updated
No Changes

### CODE REVIEW

File: changelog.json [hunk 1]
```diff
@@ -1,5 +1,15 @@
 {
   "releases": {
+    "2.9.13-beta2": [
+      "[Added] Add include and exclude functionality to selected files - #7790. Thanks @tsvetilian-ty!",
```
Comment (tidy-dev) [inline changelog.json:4]:
```suggestion
      "[Added] Add include and exclude to context menu for selected files in the changes list - #7790. Thanks @tsvetilian-ty!",
```

Comment (tidy-dev) [inline changelog.json]:
```suggestion
      "[Added] Add include and exclude to the context menu of multiple selected files in the changes list - #7790. Thanks @tsvetilian-ty!",
```

File: changelog.json [hunk 2]
```diff
@@ -1,5 +1,15 @@
 {
   "releases": {
+    "2.9.13-beta2": [
+      "[Added] Add include and exclude to the context menu of multiple selected files in the changes list - #7790. Thanks @tsvetilian-ty!",
+      "[Added] Add the option to view the current branch on GitHub - #14224. Thanks @tsvetilian-ty!",
```
Comment (tidy-dev) [inline changelog.json]:
```suggestion
      "[Added] Add \"View Branch on GitHub\" to the branches menu - #14224. Thanks @tsvetilian-ty!",
```

File: changelog.json [hunk 3]
```diff
@@ -1,5 +1,15 @@
 {
   "releases": {
+    "2.9.13-beta2": [
+      "[Added] Add include and exclude to the context menu of multiple selected files in the changes list - #7790. Thanks @tsvetilian-ty!",
```
Comment (billygriffin) [inline changelog.json]: I wanted to make this a bit more utility-focused, but we don't have to use this.
```suggestion
      "[Added] Add ability to include or exclude multiple selected files for a commit from the context menu - #7790. Thanks @tsvetilian-ty!",
```

File: changelog.json [hunk 4]
```diff
@@ -1,5 +1,15 @@
...
+      "[Added] Add \"View Branch on GitHub\" to the branches menu - #14224. Thanks @tsvetilian-ty!",
+      "[Fixed] Fix relative time of pull request review notification dialogs - #14261",
+      "[Fixed] Fix CI check status popover not closing when clicking on PR badge - #14256",
+      "[Fixed] Fix checks list overflow handling on re-run checks dialog - #14246",
+      "[Improved] Close keywords are now formatted and have informational tooltips in markdown - #14253",
+      "[Improved] Commit mentions in markdown are now formatted and linked - #14282",
+      "[Improved] Adds a link under our \"Enable notifications\" settings to the user's OS system notification settings - #14288"
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Improved] Add a link under \"Enable notifications\" settings to the user's OS system notification settings - #14288"
```

**Review by billygriffin — DISMISSED** [review]
These are optional suggestions, excited for this to go out!

**Review by billygriffin — APPROVED** [review]

**Pull request merged**